### DISCUSSION TITLE
Breeze Integration Amasty Shop by Brands

### DISCUSSION THREAD

**Discussion by waseemazam** [breezefront/community#248] [3. Integrations]
I need Breeze integration with Amasty Shop by Brands

**Comment by dimdimich** [comment]
Can you please try to contact Amasty and request integration from their side? They reported that if several clients will do that then they will consider adding integration from their side?

P.S. Also can you check if it's same or different module https://github.com/breezefront/module-breeze-amasty-shopby ?
